Cloud Application Manager/Automating Deployments/terraform-template-box.md

```diff
@@ -7,17 +7,17 @@
 }}}
```

Comment:
Please, update metadata values (date, author)

Reply:
Date updated.
I don't want to take authorship 3 lines of text.